### ISSUE TITLE
Error with a insruction given from the GUI

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1458]
In GitLab by @franra9 on Nov 20, 2024, 11:14

( This is a template for bug reports. Please fill in the relevant information and delete the rest. )

Hello @dbeltrankyl and @kinow, & @LuiggiTenorioK (?)

### Autosubmit Version

4.1.10

### Expid affected(If applicable)

a1tz

### Which task has issues? Where is the log(If applicable)

* **Full_name**:-

* **Log_Path:** ASLOGS

### Summary

change status command copy pasted from the GUI is failing

### Steps to reproduce

try changing status using: 

```
autosubmit setstatus a1tz -fl "a1tz_19900201_fc0_1_1_OPA_ENERGY_ONSHORE_6 a1tz_19900201_fc0_1_1_OPA_ENERGY_ONSHORE_7" -t COMPLETED -s -nt -np 
```

### What is the current bug behavior?

error message (` [CRITICAL] Error in the supplied input for -ft. [eCode=7011]`)

### What is the expected correct behavior?

job status successfully changed

### Relevant logs and/or screenshots(if applicable)

```
/appl/AS/4.1.10/lib64/python3.9/site-packages/paramiko-3.4.0-py3.9.egg/paramiko/pkey.py:100: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.
  "cipher": algorithms.TripleDES,
/appl/AS/4.1.10/lib64/python3.9/site-packages/paramiko-3.4.0-py3.9.egg/paramiko/transport.py:259: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.
  "class": algorithms.TripleDES,
Autosubmit is running with 4.1.10
Preparing .lock file to avoid multiple instances with same expid.

Checking configuration files...
Expdef config file is correct
Platforms sections: OK
Jobs sections OK
Autosubmit general sections OK
Configuration files OK

Loading JobList
Load finished
Creating jobs...
For the section DN with date:20240101 the number of splits is 1.
For the section APP_HYDROLAND with date:20240101 the number of splits is 1.
For the section APP_ENERGY_ONSHORE with date:20240101 the number of splits is 1.
For the section OPA_HYDROLAND_1 with date:20240101 the number of splits is 1.
For the section OPA_HYDROLAND_2 with date:20240101 the number of splits is 1.
For the section OPA_ENERGY_ONSHORE_1 with date:20240101 the number of splits is 1.
For the section OPA_ENERGY_ONSHORE_2 with date:20240101 the number of splits is 1.
For the section OPA_ENERGY_ONSHORE_3 with date:20240101 the number of splits is 1.
For the section OPA_ENERGY_ONSHORE_4 with date:20240101 the number of splits is 1.
For the section OPA_ENERGY_ONSHORE_5 with date:20240101 the number of splits is 1.
For the section OPA_ENERGY_ONSHORE_6 with date:20240101 the number of splits is 1.
For the section OPA_ENERGY_ONSHORE_7 with date:20240101 the number of splits is 1.
Adding dependencies to the graph..
Adding dependencies to the job..
Transitive reduction...
Looking for edgeless jobs...
 [CRITICAL] Error in the supplied input for -ft. [eCode=7011]
More info at https://autosubmit.readthedocs.io/en/master/troubleshooting/error-codes.html

```

### Any other relevant information(if applicable)

()

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Nov 20, 2024, 11:25

Shouldn't this be in the GUI/API repo ( in github ) ? FYI @LuiggiTenorioK 


I've checked that experiment, and I see that the datelist is 20240202, while the command is giving the date "19900201" so the job doesn't exist. 

Could it be that you recently changed the datelist, and the GUI is still not aware?

The correct command is then:

`autosubmit setstatus a1tz -fl "a1tz_20240202_fc0_1_1_OPA_ENERGY_ONSHORE_6 a1tz_20240202_fc0_1_1_OPA_ENERGY_ONSHORE_7" -t COMPLETED -s -nt -np`

**Comment by kinow** [comment]
In GitLab by @LuiggiTenorioK on Nov 20, 2024, 11:48

@franra9 How did you get the command from the GUI? I replicated the steps to get the command and I got the correct one (the same Dani put above).

**Comment by kinow** [comment]
In GitLab by @franra9 on Nov 20, 2024, 12:30

That's certainly strange.

I tried now again and it got the correct behaviour. Sorry I did not realize that they were different. For a reason I am not aware, the name of the job was not correct (Maybe just some error from my side without noticing :thinking: ) If it happens again I will report again. 

Thanks @LuiggiTenorioK and @dbeltrankyl for the fast answer!
